packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,894 @@
+    # Show the table after every command except logs and list
+    if cmd not in ('logs', 'list'):
+        print()
+        await _cmd_list(args)
+
+    return result
+
+
+# ── Command handlers ──────────────────────────────────────────────
+
+
+async def _cmd_list(args) -> int:
```

Ruff (0.15.7)

[warning] Too many branches (28 > 12)

(PLR0912)